File: sdkwork-app-sdk-csharp/Models/CheckInVO.cs
using System;
using System.Collections.Generic;
using System.Text.Json.Serialization;

namespace App.Models
{
    public class CheckInVO
    {
        public string? CreatedAt { get; set; }
        public string? UpdatedAt { get; set; }
        public bool? Success { get; set; }
        public string? CheckInDate { get; set; }
        public int? ConsecutiveDays { get; set; }
        public int? Points { get; set; }
        public Dictionary<string, object>? ExtraRewards { get; set; }
        public bool? CheckedInToday { get; set; }
        public string? Message { get; set; }
    }
}

File: sdkwork-app-sdk-go/types/check_in_vo.go
package types

// 签到结果
type CheckInVO struct {
	CreatedAt string `json:"createdAt"`
	UpdatedAt string `json:"updatedAt"`
	Success bool `json:"success"`
	CheckInDate string `json:"checkInDate"`
	ConsecutiveDays int `json:"consecutiveDays"`
	Points int `json:"points"`
	ExtraRewards map[string]interface{} `json:"extraRewards"`
	CheckedInToday bool `json:"checkedInToday"`
	Message string `json:"message"`
}
